spec/rubocop/cop/rspec/named_subject_spec.rb

```diff
@@ -0,0 +1,63 @@
+# encoding: utf-8
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::NamedSubject do
+  subject(:cop) { described_class.new }
+
+  it 'checks it for explicit subject usage' do
+    expect_violation(<<-SRC)
```

Comment:
The other cop specs use `<<-RUBY` rather than `<<-SRC`, probably worth sticking to that?